src/agent/Agent.ts

```diff
@@ -147,7 +146,9 @@ export class Agent {
       })
     }
 
-    return this.inboundTransporter.start(this)
+    if (this.inboundTransporter) {
+      await this.inboundTransporter.start(this)
+    }
```

Comment:
Maybe we should log a warning for both the inbound/outbound transporter if they are not set at initialization yet. Just for information

Reply:
Hmm, actually I can imagine it's not a problem not having an inbound transported set. But at least the info could be good. For example, I run AFJ as a mediator with Nest.js framework and it's quite convenient I don't have to set inbound transport but I handle it by myself.